services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: MovieSession = None) -> MovieSession:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> int:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: MovieSession = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    get_session = MovieSession.objects.get(id=session_id)
```

Comment:
There is no need to name this variable as `get_session`, just `session` would be enough.